tests/api/test_nanopubs.py

```diff
@@ -100,3 +99,45 @@ def test_attribute_view(self):
         self.assertEquals(json_content['label'], "Jane Doe")
         self.assertEquals(len(json_content['type']), 1)
         self.assertEquals(json_content['type'][0]['label'], 'Person')
+
+    def test_ontology_describe_view(self):
+        ontology = """
+        <http://example.com/> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2002/07/owl#Ontology> .
+        <http://example.com/janedoe> <http://schema.org/jobTitle> "Professor";
+        <http://schema.org/name> "Jane Doe" ;
+        <http://schema.org/telephone> "(425) 123-4567" ;
+        <http://schema.org/url> <http://www.janedoe.com> ;
+        <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Person> . """
+
+        self.login(*self.create_user("user@example.com", "password"))
+        response = self.client.post("/pub",
+                                    data=ontology,
+                                    content_type="text/turtle",
+                                    follow_redirects=True)
+        self.assertStatus(response, 201,
+                          "Did not receive '201 CREATED' when publishing ontology nanopub")
+
+        content = self.client.get("/about",
```

Comment:
Lines 120-125 should be factored out into a reusable method on ApiTestCase. Pass in the view and the (optional) template name.